### DISCUSSION TITLE
Homey pro - Application Iopool

### DISCUSSION THREAD

**Discussion by LucAntoine** [iopool/community#6] [Ideas]
Bonjour, j'aimerai me lancer dans le développement d'un application pour la box domotique Homey pro. Si cela intéresse du monde je veux bien un petit coup de main 😉
Merci au support de m'envoyer une clef API svp.

**Comment by mehdyouras** [comment]
Bonjour, 
bienvenu et merci d'avoir rejoint la communauté de développeurs iopool !

Au sujet de la clé d'api contactez le support directement via l'app. Ce sera plus facile pour nous de vous identifier.

**Comment by poussun-ops** [comment]
Je suis aussi utilisateur de Homey Pro (et Home Assistant) et de IoPool... si vous cherchez quelqu'un pour tester, je suis disponible :)

**Comment by LucAntoine** [comment]
Je n'ai pas pris le temps de faire une application mais je lance un script que j'ai construit pour récupérer les infos.
